VideoApi/VideoApi/Controllers/EndOfDayController.cs

```diff
@@ -0,0 +1,58 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using NSwag.Annotations;
+using VideoApi.Common.Security.Supplier.Base;
+using VideoApi.Contract.Responses;
+using VideoApi.DAL.Queries;
+using VideoApi.DAL.Queries.Core;
+using VideoApi.Domain;
+using VideoApi.Mappings;
+using VideoApi.Services;
+
+namespace VideoApi.Controllers;
+
+[Consumes("application/json")]
+[Produces("application/json")]
+[Route("end-of-day")]
+[ApiController]
+public class EndOfDayController : ControllerBase
+{
+    private readonly IQueryHandler _queryHandler;
+    private readonly SupplierConfiguration _supplierConfiguration;
+    private readonly ILogger<EndOfDayController> _logger;
+    
+    public EndOfDayController(IQueryHandler queryHandler, ILogger<EndOfDayController> logger,
+        ISupplierApiSelector supplierLocator)
+    {
+        _queryHandler = queryHandler;
+        _logger = logger;
+        _supplierConfiguration = supplierLocator.GetSupplierConfiguration();
+    }
+    
+    /// <summary>
+    /// Get all active conferences.
+    /// This includes conferences that are in progress or paused.
+    /// This includes conferences that are closed but the participants are still in consultation.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("active-sessions")]
+    [OpenApiOperation("GetActiveConferences")]
+    [ProducesResponseType(typeof(List<ConferenceForAdminResponse>), (int) HttpStatusCode.OK)]
+    public async Task<IActionResult> GetActiveConferences()
```

Comment:
just wondering if ConferenceController would be a better home for this, as it could be re-used outside the end of day scenario

Reply:
I don't suspect this would be re-used but also the controllers need to be trimmed down a bit. some of the testing is getting a bit hard to manage because of the dependencies needed for a few routes. 
We can still re-use the same routes despite grouping the operations such as gets all conferences needed for lists into one, a basic crud controller for the conference, etc